src/main/java/net/syscon/elite/api/model/PendingDeletionRequest.java

```diff
@@ -26,19 +24,22 @@
 @ApiModel(description = "PendingDeletionRequest")
 public class PendingDeletionRequest {
 
-    @ApiModelProperty(value = "An ID uniquely identifying the deletion request batch")
+    @NotNull
+    @ApiModelProperty(value = "An ID uniquely identifying the deletion request batch", example = "123", required = true)
     @JsonProperty("batchId")
     private Long batchId;
 
     @Builder.Default
-    @DateTimeFormat(iso = DATE)
-    @ApiModelProperty(value = "The start of the deletion-due-date window (defaults to epoch)")
+    @ApiModelProperty(value = "The start of the deletion-due-date window in Europe/London (ISO 8601) format without timezone offset e.g. YYYY-MM-DDTHH:MM:SS (defaults to epoch)", example = "2020-02-28T14:40:00")
     @JsonProperty("dueForDeletionWindowStart")
     private LocalDateTime dueForDeletionWindowStart = EPOCH.atStartOfDay();
 
     @NotNull
-    @DateTimeFormat(iso = DATE)
-    @ApiModelProperty(value = "The end of the deletion-due-date window")
+    @ApiModelProperty(value = "The end of the deletion-due-date window in Europe/London (ISO 8601) format without timezone offset e.g. YYYY-MM-DDTHH:MM:SS", example = "2020-02-28T14:40:00", required = true)
     @JsonProperty("dueForDeletionWindowEnd")
     private LocalDateTime dueForDeletionWindowEnd;
+
+    @ApiModelProperty(value = "The limit of the number of offenders retrieved and referred for deletion", example = "10")
+    @JsonProperty("limit")
+    private Integer limit;
```

Comment:
Would it make sense for this to have a sensible default?

Reply:
I'm purposefully allowing this to be null so that a non-limited request can still be made